src/features/Main/index.jsx

```diff
@@ -38,6 +39,7 @@ const Main = () => {
   const routes = [
     { path: '/dashboard', component: DashboardPage, exact: true },
     { path: '/students', component: StudentsPage, exact: true },
+    { path: '/students/:learnerEmail', component: StudentsDetails, exact: true },
```

Comment:
Please check the cases when there are special characters, for example: student/@example.com is a valid email for the platform

Reply:
Fixed